src/components/simple/static-api.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div>
+    <h2>Absolute URL Static API Example</h2>
+    // partial URL gets domain from client
```

Comment:
this super nitpicky but did you mean to make this an html comment (like below) or did you want these to render as text on the test app?
```suggestion
    <!–– partial URL gets domain from client -->
```